apis/management.cattle.io/v3/authn_types.go

```diff
@@ -146,8 +146,44 @@ type ActiveDirectoryConfig struct {
 }
 
 type ActiveDirectoryTestAndApplyInput struct {
-	ActiveDirectoryConfig ActiveDirectoryConfig `json:"activeDirectoryConfig, omitempty"`
+	ActiveDirectoryConfig ActiveDirectoryConfig `json:"activeDirectoryConfig,omitempty"`
 	Username              string                `json:"username"`
 	Password              string                `json:"password"`
 	Enabled               bool                  `json:"enabled,omitempty"`
 }
+
+type OpenLDAPConfig struct {
```

Comment:
Where is loginDomain? it was a field in v1.6. I don't see it here. It's called DefaultLoginDomain on ActiveDirectory